### PR TITLE
Updated the version of azure management sdks.

### PR DESCRIPTION
**Pull request by sarangan12**
Updated the version of azure sdks.

@veronicagg @vishrutshah Please review.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,8 +13,9 @@ group :development, :test do
...
-gem 'azure_mgmt_network',   '~>0.5.0'
-gem 'azure_mgmt_storage',   '~>0.5.0'
-gem 'haikunator',           '~>1.1'
\ No newline at end of file
+gem 'azure_mgmt_resources', '~>0.6.0'
+gem 'azure_mgmt_compute',   '~>0.6.0'
+gem 'azure_mgmt_network',   '~>0.6.0'
+gem 'azure_mgmt_storage',   '~>0.6.0'
+gem 'haikunator',           '~>1.1'
+
```
Comment (vishrutshah) [inline Gemfile]: minor: new empty line is unnecessary.

Comment (sarangan12) [inline Gemfile]: It has been a practice in cordova/TS projects to ensure that there is a new line at the end. Actually it is one of the best practices in coding. One sample discussion on this: http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Comment (vishrutshah) [inline Gemfile]: That's good to know. Thanks for the link. It totally makes sense now 👍

Comment (vishrutshah) [inline Gemfile]: @sarangan12 With your comments i got very interested in understanding this idea. I found an interesting thing.

GitHub itself trails the last trailing line if the file has it or shows some symbol. Now in your changes GitHub is showing one empty line that implies that you must have two empty line in this file at the end. It'd be good if you can remove extra unnecessary line. Let me know if i am misunderstood. 

Thanks for helping me understand this :)

Comment (sarangan12) [inline Gemfile]: @vishrutshah I think Github may do this in Github Desktop. Did you check using Github Desktop? Normally you can see the red marker in the PR if there is no end of line.

Comment (vishrutshah) [inline Gemfile]: That's strange. I tried using checking out your branch and seeing this file in RubyMine. I can see two lines. Let me know if you are using RubyMine and it doesn't show two lines then it must be some of my local configurations.

Comment (vishrutshah) [inline Gemfile]: Here is the interesting detailes: I used RuboCop to find out what should be the expected behaviors for the line endings. Here is the `.rubocop` configuration [Reference: [.rubocop.yml](https://github.com/rails/rails/blob/master/.rubocop.yml#L27)]

```
# No extra empty lines.
Style/EmptyLines:
  Enabled: true
```

**Current condition on the master - Gemfile**
master_branch

**Condition with your PR branch - Gemfile**
sarangan12

**Condition with your PR branch removing extra new line - Gemfile**
deleteline

Rubocop sees extra new line (2 CR LF) as offense on having an extra line but not when 1 CR LF feed at the end of file. It's really interesting. What do you think?

Comment (sarangan12) [inline Gemfile]: That is interesting. There seems to be difference with GitHub - Robocop - Vim - Sublime handles the situations. (Dont forget Atlassaian!!!! Its sourcetree actually complains for missing a new line at the end :-) Let us explore this offline)

**Pull request merged**